src/app/src/main/kotlin/org/imperial/mrc/comet/controllers/DataController.kt

```diff
@@ -13,6 +14,12 @@ class DataController(
     private val logger: Logger = LoggerFactory.getLogger(DataController::class.java),
     private val apiClient: APIClient
 ) {
+    @GetMapping("/countries")
+    fun countries(): ResponseEntity<String> {
+        logger.info("countries")
+        return apiClient.countries()
```

Comment:
I noticed that this endpoint returned a "TEST" country. We'll presumably remove this before final release.

Reply:
Ah good spot, yes - or we'll need to filter out countries whose `public` flag is false, either in front or back end. To be clarified!